utils/prometheus/prometheus.yaml

```diff
@@ -31,11 +31,10 @@ scrape_configs:
       - targets:
           - "pub:7105"
 
-  - job_name: fatman-v-0-0-1
+  - job_name: racetrack-job
     scheme: http
     metrics_path: /metrics
-    dns_sd_configs:
-      - names: 
-          - 'fatman-adder-v-0-0-1'
-        type: A
-        port: 7000
+    file_sd_configs:
+      - files:
+        - '/mnt/plugins/metrics/sd_config_jobs.yaml'
+        refresh_interval: 1m
```

Comment:
[nitpick]This single entry will serve all jobs that are deployed, right? Then naming it `racetrack-jobs` would make it more clear what it's doing. [/nitpick]

Reply:
Right, renaming makes sense